test/archetype.test.ts

```diff
@@ -1679,18 +1731,161 @@ describe("Factory", function () {
       start: ethers.BigNumber.from(Math.floor(Date.now() / 1000)),
       end: 0,
       limit: 40,
-      maxSupply: DEFAULT_CONFIG.maxSupply,
+      maxSupply: default_config.maxSupply,
       unitSize: 0,
+      randomize: true,
+      tokenIds: [],
       tokenAddress: ZERO,
     });
 
     await nftMint.connect(minter2).mint({ key: HASH256, proof: [] }, 40, ZERO, "0x", { value: 0 });
 
-    await expect(await nftMint.totalSupply()).to.be.equal(DEFAULT_CONFIG.maxSupply);
+    await expect(await nftMint.totalSupply()).to.be.equal(100);
+  });
+
+  it("test erc115 specific tokenId mints", async function () {
```

Comment:
fix typo. should be erc1155 not erc115.
actually tripped me up because I was searching for the 1155 tests :)